deeplabcut/utils/make_labeled_video.py

```diff
@@ -152,12 +156,36 @@ def CreateVideo(
         C = colorclass.to_rgba(np.linspace(0, 1, nindividuals))
     colors = (C[:, :3] * 255).astype(np.uint8)
 
+    bboxes_color = (0, 0, 0)
+
     with np.errstate(invalid="ignore"):
-        for index in trange(min(nframes, len(Dataframe))):
+        for index in trange(min(nframes, len(dataframe))):
             image = clip.load_frame()
             if displaycropped:
                 image = image[y1:y2, x1:x2]
 
+            # Draw bounding boxes if required and present
+            if plot_bboxes and bboxes_list:
+                bboxes      = bboxes_list[index]["bboxes"]
+                bbox_scores = bboxes_list[index]["bbox_scores"]
+                n_bboxes = bboxes.shape[0]
+                for i in range(n_bboxes):
+                    bbox = bboxes[i, :]
+                    x, y = bbox[0], bbox[1]
+                    x += x1
+                    y += y1
+                    w, h = bbox[2], bbox[3]
+                    confidence = bbox_scores[i]
+                    if confidence < bboxes_pcutoff:
+                        continue
+                    rect_coords = rectangle_perimeter(start=(y, x), extent=(h, w))
+
+                    set_color(
```

Comment:
I was wondering why we are using two different libraries (`pyplot` or `skimage`) to mark images? These libraries are used to do basically the same thing, but the produced results look different (for example the added markers shapes look different, the line widths of rectangles are not the same...).
Pyplot seems to offer richer features (we can do dashed lines for examples, different lines widths, etc), so would be great if we could switch completely to it...

Do we need both because `skimage` works faster on videos (fastmode) than `pyplot`? But why do we keep pyplot then? Is it to have different labels (+, x, .) for network evaluation for example?